modules/ROOT/assets/attachments/sync-gateway-admin.yaml

```diff
@@ -265,7 +265,7 @@ parameters:
   since:
     in: query
     name: since
-    description: Starts the results from the change immediately after the given sequence ID. Sequence IDs should be considered opaque; they come from the last_seq property of a prior response.
+    description: Starts the results from the change immediately after the given sequence ID. Sequence IDs can be integer or dictionary types and should be considered opaque; they come from the last_seq property of a prior response.
```

Comment:
I think it might be best to just leave it as 'Sequence IDs should be considered opaque', and change the type to 'object' everywhere.  Today they can be integer or JSON String - I don't think there's a case where we send dictionary - but I agree it's correct they should be considered an opaque object.